packages/grid/src/vaadin-grid-sort-mixin.js

```diff
@@ -87,7 +87,9 @@ export const SortMixin = (superClass) =>
 
     /** @private */
     __updateSortOrders() {
-      this._sorters.forEach((sorter, index) => (sorter._order = this._sorters.length > 1 ? index : null), this);
+      this._sorters.forEach((sorter, index) => {
+        sorter._order = this._sorters.length > 1 ? index : null;
+      }, this);
```

Comment:
Passing `thisArg` to `forEach` here appears to be a rudiment from Polymer 1 and ES5 ages 😄 
Somehow it was preserved when [changing codebase to ES2015](https://github.com/vaadin/vaadin-grid/pull/984/files#diff-4a3e9d67aba0772aae6a6daf63867cf2b785c95caf041efe3cb16f6906537a8fR73). Let's remove it.

BTW there are a few other cases in `vaadin-infinite-scroller`, could remove those as well.

Reply:
Done.